mkosi

```diff
@@ -1283,6 +1283,17 @@ def patch_file(filepath: str, line_rewriter: Callable[[str], str]) -> None:
     shutil.move(temp_new_filepath, filepath)
 
 
+def disable_pam_securetty(workspace: str,
+                          file_path: str = 'root/etc/pam.d/login') -> None:
```

Comment:
No reason to make the path configurable until we actually have a usecase where we need a different path. I'd just remove the parameter until we actually need it.

Reply:
I would answer "no reason to remove it until we are sure all distributions have to same conf file for that", but I trust you about it, so, changed also.